pandas/tests/indexes/test_base.py

```diff
@@ -766,6 +766,26 @@ def test_sub(self):
         self.assertRaises(TypeError, lambda: idx - idx.tolist())
         self.assertRaises(TypeError, lambda: idx.tolist() - idx)
 
+    def test_map_identity_mapping(self):
+        for name, cur_index in self.indices.items():
```

Comment:
can you add a comment with the appropriate github issue numbers (don't go crazy but where appropriate)